### PR TITLE
Feature/marshall/create user model

### PR DESCRIPTION
**Pull request by uimarshall**

### CODE REVIEW

**Review by olajohn-ajiboye — COMMENTED** [review]
All looks good with this. I think it can be merged.

File: test/connection.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+before(done => {
+    // connect to db
+    const db = require('../config/keys').mongoURI;
+    mongoose.connect(db);
+    mongoose.connection
+        .once('open', () => {
+            console.log('Connection made');
+            done();
+        })
+        .on('error', error => {
```
[the remark this answers is not in the document]
Comment (olajohn-ajiboye) [reply test/connection.js]: Remove console.logs()

**Review by olajohn-ajiboye — CHANGES_REQUESTED** [review]
Remove all console.log()

**Review by enyeInc — APPROVED** [review]
LGTM

**Pull request merged**